src/components/select/select.js

```diff
@@ -149,6 +154,30 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $interpolate) {
         }
       });
 
+      // Create a fake select to find out the label value
+      function setInitialLabelValue() {
+        if (ngModel && scope[attr.ngModel] !== undefined) {
+          var fakeSelectEl = angular.element(selectTemplate).find('md-select-menu');
+          fakeSelectEl.data('$ngModelCtrl', ngModel);
+          fakeSelectEl = $compile(fakeSelectEl)(scope.$new());
+          var fakeSelectCtrl = fakeSelectEl.controller('mdSelectMenu');
+          var fakeSelectScope = fakeSelectEl.scope();
```

Comment:
Don't use `.scope()`, it's broken when `$compileProvider.debugInfoEnabled(false)` is set, which many people do during production.

Let's just create a scope property on the controller.